openvino2tensorflow/openvino2tensorflow.py

```diff
@@ -1340,7 +1340,7 @@ def layer_structure_print(info: dict) -> None:
                                     shared_axes=shared_axes
                                 )(tf_layers_dict[get_tf_edges_from(tf_edges, layer_id, 0)])
```

Comment:
I think we also need to transpose temp_alpha for L1312 and L1318 and L1327 and L1334. This is because the same process is used when `replace_prelu_and_minmax` is invalid (False). Only L1339 is a branch to recover unexpected processing, so I think no modification is necessary.
https://github.com/PINTO0309/openvino2tensorflow/pull/106/files#diff-906bc49b3eeba462117c0cfaecd3c380a2309c9a20707ddcb200b4351d3eb63cR1312-R1334

Reply:
Hi, I added the corresponding modification to the cases where replace_prelu_and_minmax is False.